### DISCUSSION TITLE
docker support for CDT broken?

### DISCUSSION THREAD

**Discussion by ThomasMcLeod** [eclipse-cdt/cdt#883] [General]
Hello,
I'm not a very active member here, but we do have a team of engineers still using CDT for development. We roll our own build system using gmake/g++ on Ubuntu to build a 1M+ LoC project. Some team members would like to keep using Eclipse for editing, debugging, static analysis and to launch gmake using Project Explorer build targets. But we recently starting building in docker containers, and no one can figure out how to call gmake in a docker container from Eclipse. Is this possible, or was it once possible but now broken? In console says it's running in the correct image, and the correct make command is displayed, but then we get an inexplicable java error beginning with 'java -jar agnt.jar [options...] <secret key> <agent name>'. We don't use java anywhere, so I assume this is something broken in Eclipse. Please just it me know if this is possible, before my dev ops manager permanently nixes Eclipse.

**Comment by ThomasMcLeod** [comment]
The above java error seems to be related to 'Jenkins Remoting,' but we're not trying to build with Jenkins, though there is a Jenkins script present in the code tree. Our devops uses Jenkins, but engineers just want to run make inside the container.

**Comment by nitind** [comment]
Without concrete details on how you're launching (failing to launch?) the container, it's a bit difficult to say either way. You didn't include a full error message nor steps to reproduce.
